lib/src/components/VernaForm/index.tsx

```diff
@@ -42,7 +43,9 @@ function VernaForm({ onSubmit }: VernaFormProperties) {
       transformErrors={transformErrors}
       uiSchema={uiSchema}
       widgets={widgets}
-    />
+    >
+      {SubmitButton}
```

Comment:
```suggestion
      <SubmitButton />
```

Reply:
I can't, this one is a ReactNode and not a component, it's so it can be undefined in a clean way, otherwise i would have (it needs to be nothing if none is given otherwise it will still override the render of the button
```suggestion
      {SubmitButton && <SubmitButton/>}
```